pyproject.toml

```diff
@@ -1,2 +1,45 @@
 [build-system]
-requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
+requires = ["setuptools>=64.0", "setuptools_scm[toml]>=8.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mkdocs-asf-theme"
+requires-python = ">=3.8"
+authors = [
+    {name="ASF APD/Tools Team", email="uaf-asf-apd@alaska.edu"},
+]
+description = "An extension of the MkDocs Material Theme for ASF"
+readme = "README.md"
+license = {text = "BSD-3-Clause"}
+classifiers=[
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+dependencies = [
+    "mkdocs",
+    "mkdocs-material>=9.4.2,<10",
+]
+dynamic = ["version"]
+
+[project.entry-points."mkdocs.themes"]
+asf-theme = "asf_tools"
```

Comment:
Following the pattern in mkdocs-material:
https://github.com/squidfunk/mkdocs-material/blob/9.4.2/pyproject.toml#L83-L84